spec/controllers/discussions_controller_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:exercise_params) { {debatable_class: 'Exercise', exercise_id: exercise.id} }
 
   before { set_current_user! user }
+  before { Organization.current.tap { |it| it.forum_enabled = true }.save! }
```

Comment:
You could use .update!(forum_enabled: true) to avoid the extra block.
Apart from this details, LGTM :+1:  :+1:  :+1: